src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,15 @@
 package core.basesyntax;
 
 public class MainApp {
-
+    public static void main(String[] args) {
+        Truck truck = new Truck();
+        Bulldozer bulldozer = new Bulldozer();
+        Excavator excavator = new Excavator();
+        truck.doWork();
+        bulldozer.doWork();
+        excavator.doWork();
+        truck.stopWork();
+        bulldozer.stopWork();
+        excavator.stopWork();
```

Comment:
checklist: [Use abstract reference instead of specific one where possible:](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/oop#use-abstract-reference-instead-of-specific-one-where-possible). Lets create array of machines and start them and spot in a loop